packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/SortableTable.3.doc.mdx

```diff
@@ -0,0 +1,48 @@
+import { Badge } from "terra-table/package.json?dev-site-package";
+import SortableTable from "./SortableTable?dev-site-example";
+
+import TablePropsTable from "terra-table/lib/Table?dev-site-props-table";
+
+<Badge />
+
+# Sortable Table
+
+Terra Table is a structural component used to create data tables. Table provides means to handle row selection and hooks for sortable columns.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-table`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+| --------------- | ------- |
+| react           | ^16.8.5 |
+| react-dom       | ^16.8.5 |
+| react-intl      | ^2.8.0  |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+
+```jsx
+import Table from "terra-table";
+```
+
+## Component Features
+
+- [Cross-Browser Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#cross-browser-support)
+- [Responsive Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#responsive-support)
+- [Mobile Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#mobile-support)
+- [LTR/RTL Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#ltr--rtl)
+
+<SortableTable />
+
+## Table Props
+
+<TablePropsTable />
```

Comment:
Your examples should not have the property table for the Terra Table or its dependencies.  That would be part of the main About document.

Reply:
Removed from all examples.